communities/src/lib.rs

```diff
@@ -128,7 +155,14 @@ decl_event!(
     where
         AccountId = <T as frame_system::Config>::AccountId,
     {
+        /// A new community was registered \[who, community_identifier\]
         CommunityRegistered(AccountId, CommunityIdentifier),
+        /// CommunityMetadata was updated \[who, community_identifier\]
+        MetadataUpdated(CommunityIdentifier),
+        /// A community's nominal income was updated \[who, community_identifier\]
+        NominalIncomeUpdated(CommunityIdentifier),
```

Comment:
it would be lightweight to add the new value here

Reply:
Of what value are you speaking?

Reply:
ah ok, I got it now!